src/sync-system.js

```diff
@@ -84,5 +88,29 @@ AFRAME.registerSystem('sync-system',
 				get: function () { return masterClientId === this.clientId; }.bind(this)
 			});
 		}.bind(this));
-	}
+	},
+	instantiateRemotely: function (el, components, removeOnDisconnect) {
+		var ref = this.remoteElementsRef.push({clientId: this.clientId, type: el.nodeName, components: components});
+		if (removeOnDisconnect) {
+			ref.onDisconnect().remove();
+		}
+		return ref;
+	},
+	createElement: function (snapshot) {
+		var elInfo = snapshot.val();
+		if (!elInfo || !elInfo.type) { return; }
+		if (elInfo.clientId === this.clientId) { return; }
```

Comment:
a way of projecting past the current case here would be to have the instantiator dunce hat be actually spawned on the head by an object that isn't the hat and changing the color on it